### ISSUE TITLE
Improve the documentation for the function

### ISSUE THREAD

**Issue by vuoanh** [MolSSI-SSS/QM_2017_SSS_Team10#52]
Could you guys update the documentation so that it explains the data types and the meaning of the parameters?

**Comment by bgutier5** [comment]
Would you like to know the data types of the arguments, the outputs or both?

**Comment by vuoanh** [comment]
yes, it would be very nice if you could do that for the outputs too!
